python/kfserving/docs/KFServingClient.md

```diff
@@ -10,6 +10,7 @@ config_file | Name of the kube-config file. Defaults to `~/.kube/config`. |
 context |Set the active context. If is set to None, current_context from config file will be used.|
 client_configuration | The kubernetes.client.Configuration to set configs to.|
 persist_config | If True, config file will be updated when changed (e.g GCP token refresh).|
+load_kube_config | Set to `True` if the SDK is running in the cluster, and want to operate KFService in another cluster. Defaults to `False`. |
```

Comment:
why do we need another variable? of config file is specified, load it. If not, use assuming its the default one..?

Reply:
@animeshsingh See the PR https://github.com/kubeflow/kfserving/pull/390 for details. The old logic is : only if NOT running in the cluster,  load the config file. After the variable, even if in the cluster, we also can load other cluster config.
I used that while doing E2E (we created new cluster, and install istio/knative/kfserving), and test-worker still is running in cluster, that's OK if using the `load_kube_config`.

Reply:
I understand - but the old logic itself isn't flawed? is there a cost to be paid for loading? if not, just load? If a path is specified we load from there, if not. we assume its default?

Reply:
In fact, that's a enhancement. Just loading, and user can specify the conf path or use default value, that's optional. As used here:
https://github.com/kubeflow/kfserving/blob/836f3534da0ed5551c035d54df18ba68f51220d4/test/e2e/test_tensorflow_inference_service.py#L30

Reply:
so instead of doing this
KFServing = KFServingClient(load_kube_config=True)

I do 
KFServing = KFServingClient(/path/to/kube_config)
This tells the system I want to load, and hence am providing the path? Why do we need another variable?

Reply:
@animeshsingh That's what I think too :-) Updated. Thanks.